ivy/functional/frontends/torch/func_wrapper.py

```diff
@@ -215,8 +255,6 @@ def outputs_to_frontend_arrays_torch(*args, **kwargs):
 
 
 def to_ivy_arrays_and_back(fn: Callable) -> Callable:
-    """Wrap `fn` so that input arrays are all converted to `ivy.Array` instances and
-    return arrays are all converted to `Tensor` instances."""
```

Comment:
Is there a specific reason you removed the docstrings? I think it's useful to have a brief explanation of what each decorator does.

Reply:
Other than that, everything looks good!

Reply:
What if you put both `"""` in separate lines?

Reply:
I skipped the pre-commit hook and finaly manged to commit. Thanks.